_config.yml

```diff
@@ -1,18 +0,0 @@
-# Jekyll is only used for documentation at this point
-layouts_dir:  ./documentation/_layouts
-data_dir:     ./documentation/_data
-includes_dir: ./documentation/_includes
-
-# Enable Github flavored markdown
-kramdown:
-  input: GFM
-  hard_wrap: false
-
-# Set default values for documentation
-defaults:
-  -
-    scope:
-      path: "documentation"
```

Comment:
I left the contents of the `documentation` folder intact, all of them.  We may use them at a later date.  We may even put together a real `/docs` endpoint.  Now that we have a big boy server, we can do those kinds of things 😉